### PR TITLE
Checklist

### PR DESCRIPTION
**Pull request by LucsT**
Doing checklist for production use

### CODE REVIEW

**Comment by laedit** [comment]
@LucsT sounds good to me, do you need me to merge it or do you prefer that a member of cozy team review it first, since it is mainly cozy-related modifications?

File: package.json [hunk 1]
```diff
@@ -4,32 +4,43 @@
...
     "dev": "tsc && cozy-run-dev",
     "standalone": "tsc && cozy-run-standalone",
     "build": "webpack",
-    "lint": "standard --fix konnector.js",
-    "deploy:travis": "git-directory-deploy --username laedit --email $EMAIL --directory build/ --repo=https://$GITHUB_TOKEN@github.com/laedit/cozy-konnector-theoldreader.git",
-    "deploy": "git-directory-deploy --directory build/ --branch build --repo=https://github.com/laedit/cozy-konnector-theoldreader.git"
+    "precommit": "yarn lint",
+    "lint": "eslint . --fix",
+    "deploy": "git-directory-deploy --directory build/ --branch build --repo=${DEPLOY_REPOSITORY:-https://$GITHUB_TOKEN@github.com/konnectors/cozy-konnector-theoldreader.git}",
+    "deploy:prod": "git-directory-deploy --directory build/ --branch latest --repo=${DEPLOY_REPOSITORY:-https://$GITHUB_TOKEN@github.com/konnectors/cozy-konnector-theoldreader.git}"
```
Comment (kosssi) [inline package.json]: Il y a eu un changement sur le deploy tu peux voir ce commit : https://github.com/konnectors/cozy-konnector-template/commit/e06d6cbc162adaee34a342de99a72ebe91d70a83

Comment (LucsT) [reply package.json]: @kossi, c'est fait.

Comment (LucsT) [reply package.json]: J'ai amend le dernier commit avec la modif tu .travis.yml

Comment (LucsT) [reply package.json]: @kossi

**Pull request merged**